src/runtime/providers/prismic.ts

```diff
@@ -0,0 +1,36 @@
+import type { ProviderGetImage } from 'src'
+import { $URL, joinURL } from 'ufo'
+import { operationsGenerator } from './imgix'
+
+const PRISMIC_IMGIX_BUCKET = 'https://images.prismic.io'
+
+// Prismic image bucket is left configurable in order to test on other environments
+export const getImage: ProviderGetImage = (
+  src,
+  { modifiers = {}, baseURL = PRISMIC_IMGIX_BUCKET } = {}
+) => {
+  const operations = operationsGenerator(modifiers)
+
+  // If there are already some query parameters set by Prismic
+  const joinOperator = src.includes('?') ? '&' : '?'
```

Comment:
We can use parseURL/parseQuery utilities to extract pathname and query

Reply:
Updated to those utilities, UFO's readme got me confused when doing this first~